src/main/java/org/sunbird/profile/service/UserBulkUploadService.java

```diff
@@ -417,6 +423,273 @@ private void processBulkUpload(HashMap<String, String> inputDataMap) throws IOEx
+                            invalidErrList.add("Invalid Mother Tongue: Mother Tongue should be added from default list and/or cannot contain special character");
+                        }
+                    }
+
+                   if (!isFieldEmpty(record, 9)) {
+                        String employeeId = record.get(9).trim();
+                        userRegistration.setEmployeeId(employeeId);
+                        if (!ProjectUtil.validateEmployeeId(userRegistration.getEmployeeId())) {
+                            invalidErrList.add("Invalid Employee ID : Employee ID can contain alphanumeric characters or numeric character and have a max length of 30");
+                        }
+                        if (userRegistration.getEmployeeId().contains(Constants.SPACE)) {
+                            invalidErrList.add("Invalid Employee ID : Employee Id cannot contain spaces");
+                        }
+                    }
+
+                    if (!isFieldEmpty(record, 10)) {
+                        String pincode = record.get(10).trim();
+                        userRegistration.setPincode(pincode);
+                        if (!ProjectUtil.validatePinCode(userRegistration.getPincode())) {
+                            invalidErrList.add("Invalid Office Pin Code : Office Pin Code should be numeric and is of 6 digit.");
+                        }
+                    }
+
+                    if (!isFieldEmpty(record, 11)) {
+                        String externalSystemId = record.get(11).trim();
+                        userRegistration.setExternalSystemId(externalSystemId);
+                        if (!ProjectUtil.validateExternalSystemId(userRegistration.getExternalSystemId())) {
+                            invalidErrList.add("Invalid External System ID : External System Id can contain alphanumeric characters and have a max length of 30");
+                        }
+                    }
+                    if (!isFieldEmpty(record, 12)) {
+                        String externalSystem = record.get(12).trim();
+                        userRegistration.setExternalSystem(externalSystem);
+                        if (!ProjectUtil.validateExternalSystem(userRegistration.getExternalSystem())) {
+                            invalidErrList.add("Invalid External System Name : External System Name can contain only alphabets and alphanumeric and can have a max length of 255");
+                        }
+                    }
+
+                    if (!isFieldEmpty(record, 13)) {
+                        String tagStr = record.get(13).trim();
+                        List<String> tagList = new ArrayList<>();
+                        if (!StringUtils.isEmpty(tagStr)) {
+                            String[] tagStrList = tagStr.split(",", -1);
```

Comment:
Let's expect TAG value to be given like - Test1 & Test2
Split the TAG using "&" and trim it...